tests/suites/test_suite_x509parse.data

```diff
@@ -907,7 +907,7 @@ X509 CRT ASN1 (Empty Certificate)
 x509parse_crt:"":"":MBEDTLS_ERR_X509_INVALID_FORMAT
 
 X509 CRT ASN1 (inv Certificate, bad tag)
-x509parse_crt:"00":"":MBEDTLS_ERR_X509_INVALID_FORMAT
+x509parse_crt:"0500":"":MBEDTLS_ERR_X509_INVALID_FORMAT
```

Comment:
Aren't we reducing test coverage by only testing with ASN.1 structures that are well-formed? Shouldn't we still continue testing with ill-formed ASN.1 structures?

Reply:
We are testing various ill-formed structures, too, but for the purpose of testing the behavior of the parser on invalid tags, it's ok to keep a well-formed structure.